internal/cli/atlas/livemigrations/cutover.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package livemigrations
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var createTemplate = "Cutover process '{{.ID}}' successfully started.\n"
+
+type CutoverOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store           store.LiveMigrationCutoverStarter
+	liveMigrationID string
+}
+
+func (opts *CutoverOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+func (opts *CutoverOpts) Run() error {
+	r, err := opts.store.StartLiveMigrationCutover(opts.ConfigOrgID(), opts.liveMigrationID)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas liveMigrations|lm cutover [--liveMigrationID liveMigrationId] [--projectId projectId].
+func CutoverBuild() *cobra.Command {
+	opts := &CutoverOpts{}
+	cmd := &cobra.Command{
+		Use:   "cutover",
+		Short: "Starts the process to stop the data transmission between clusters.",
```

Comment:
Let's go with "initiates" since "start to stop" seems funny 😂 
```suggestion
		Short: "Initiates the process to stop data transmission between clusters.",
		Long:  "Your API Key must have the Organization Owner role to successfully run this command.",
```